hw/xbox/nv2a/nv2a_psh.c

```diff
@@ -593,43 +645,58 @@ static QString* psh_convert(struct PixelShader *ps)
     ps->code = qstring_new();
 
     for (i = 0; i < 4; i++) {
-
         const char *sampler_type = NULL;
 
         switch (ps->tex_modes[i]) {
         case PS_TEXTUREMODES_NONE:
-            qstring_append_fmt(vars, "vec4 t%d = vec4(0.0); /* PS_TEXTUREMODES_NONE */\n",
-                               i);
+            qstring_append_fmt(
+                vars, "vec4 t%d = vec4(0.0); /* PS_TEXTUREMODES_NONE */\n", i);
             break;
         case PS_TEXTUREMODES_PROJECT2D:
             if (ps->state.rect_tex[i]) {
                 sampler_type = "sampler2DRect";
             } else {
                 sampler_type = "sampler2D";
             }
-            qstring_append_fmt(vars, "vec4 t%d = textureProj(texSamp%d, pT%d.xyw);\n",
-                               i, i, i);
+            qstring_append_fmt(vars,
+                               "vec4 t%d = textureProj(texSamp%d, pT%d.xyw);\n",
+                               i,
+                               i,
+                               i);
             break;
         case PS_TEXTUREMODES_PROJECT3D:
             sampler_type = "sampler3D";
-            qstring_append_fmt(vars, "vec4 t%d = textureProj(texSamp%d, pT%d.xyzw);\n",
-                               i, i, i);
+            qstring_append_fmt(
+                vars,
+                "vec4 t%d = textureProj(texSamp%d, pT%d.xyzw);\n",
+                i,
+                i,
+                i);
             break;
         case PS_TEXTUREMODES_CUBEMAP:
             sampler_type = "samplerCube";
-            qstring_append_fmt(vars, "vec4 t%d = texture(texSamp%d, pT%d.xyz / pT%d.w);\n",
-                               i, i, i, i);
+            qstring_append_fmt(
+                vars,
+                "vec4 t%d = texture(texSamp%d, pT%d.xyz / pT%d.w);\n",
+                i,
+                i,
+                i,
+                i);
             break;
         case PS_TEXTUREMODES_PASSTHRU:
             qstring_append_fmt(vars, "vec4 t%d = pT%d;\n", i, i);
             break;
         case PS_TEXTUREMODES_CLIPPLANE: {
             int j;
-            qstring_append_fmt(vars, "vec4 t%d = vec4(0.0); /* PS_TEXTUREMODES_CLIPPLANE */\n",
-                               i);
+            qstring_append_fmt(
+                vars,
+                "vec4 t%d = vec4(0.0); /* PS_TEXTUREMODES_CLIPPLANE */\n",
+                i);
             for (j = 0; j < 4; j++) {
-                qstring_append_fmt(vars, "  if(pT%d.%c %s 0.0) { discard; };\n",
-                                   i, "xyzw"[j],
+                qstring_append_fmt(vars,
+                                   "  if(pT%d.%c %s 0.0) { discard; };\n",
+                                   i,
+                                   "xyzw"[j],
```

Comment:
swizzlemask (and writemasks) should always be on same line as register / variable they belong to in my opinion.